adapters/ethereum/index.js

```diff
@@ -180,6 +178,16 @@ function toEthereumChannel(channel) {
 	})
 }
 
+function getCoreContractForChannel(cfg, channel, ethProvider) {
+	const depositChainId =
+		channel.spec.depositChainId || cfg.depositAssetsToChainId[channel.depositAsset]
+	const provider =
+		ethProvider ||
+		(depositChainId && new providers.JsonRpcProvider(cfg.supportedChainIdsByRPC[depositChainId])) ||
+		getDefaultProvider(cfg.ETHEREUM_NETWORK)
```

Comment:
there is no `ETHEREUM_NETWORK` in the cfg - it is `NETWORK`. In `docs/configuration.md`  it's `ETHEREUM_NETWORK` - just has to be `NETWORK` everywhere I guess.